ci/tests/vpool/vpool_test.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2015 iNuron NV
+#
+# Licensed under the Open vStorage Non-Commercial License, Version 1.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.openvstorage.org/OVS_NON_COMMERCIAL
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ci.tests.general import general
+from ci.tests.general.general import test_config
+from ci.tests.backend import alba, generic
+from ovs.dal.lists.backendlist import BackendList
+from ovs.dal.lists.albanodelist import AlbaNodeList
+from ovs.lib.albacontroller import AlbaController
+from ovs.lib.albanodecontroller import AlbaNodeController
+from ovs.lib.storagerouter import StorageRouterController
+from ovs.dal.lists.vpoollist import VPoolList
+from ci.tests.disklayout import disklayout
+
+VPOOL_NAME = test_config.get('vpool', 'vpool_name')
+VPOOL_NAME = 'vpool-' + VPOOL_NAME
+BACKEND_NAME = test_config.get('backend', 'name')
+BACKEND_TYPE = test_config.get('backend', 'type')
+GRID_IP = test_config.get('main', 'grid_ip')
+
+
+def setup():
+    disklayout.add_db_role()
+    disklayout.add_read_write_scrub_roles()
+    if not generic.is_backend_present(BACKEND_NAME, BACKEND_TYPE):
+        backend_guid = alba.add_alba_backend(BACKEND_NAME)
+    backend = BackendList.get_by_name(BACKEND_NAME)
+    alba_node = AlbaNodeList.get_albanode_by_ip(GRID_IP)
+    # claim disks up to max
+
```

Comment:
the claim disks part is alba related, so I'd suggest moving this code to ci/tests/backend/alba.py